### PR TITLE
feat: Add support for Elm style comment syntax

### PR DESCRIPTION
**Pull request by mattapperson**
Hey! Thanks for building this plugin!

I created a quick PR to support Elm Lang based repos too + added tests for it.

Let me know if there is anything else you would like to see in this PR :)

### CODE REVIEW

**Comment by rohit-gohri** [comment]
This is great! Glad someone found this plugin useful :smiley:

File: src/index.ts [hunk 1]
```diff
@@ -79,7 +79,7 @@ export function getFormattedSrcLink(filepath: string, repoUrl?: RepoUrl) {
 export default async function todos({
   repoUrl = true,
   ignore = [],
-  keywords = ["TODO", "FIXME"],
+  keywords = ["TODO", "FIXME", "BUGBUG"],
```
Comment (rohit-gohri) [inline src/index.ts]: Just one nitpick, is 'BUGBUG' a common pattern? It's the first time I'm seeing it. So if it's something that you personally use then I'd prefer not adding it to defaults.

Comment (mattapperson) [reply src/index.ts]:
```suggestion
  keywords = ["TODO", "FIXME"],
```

**Comment by mattapperson** [comment]
Resolved :)

**Review by rohit-gohri — APPROVED** [review]

**Pull request merged**